prismic-model/js/parts/paragraph.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+export default function paragraph(label: string = 'Text') {
```

Comment:
Could we just use structured text? Trying to not put that many other types on top of the standard types:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/prismic-model/js/parts/structured-text.js

Reply:
Shoudl we?